core-java/src/main/java/com/baeldung/instrumentation/agent/MyInstrumentationAgent.java

```diff
@@ -34,7 +34,7 @@ private static void transformClass(String className, Instrumentation instrumenta
         } catch (Exception ex) {
             LOGGER.error("Class [{}] not found with Class.forName");
         }
-        // otherwise iterate all loaded classes and find what we want
+        // otherwise iterate all loaded className and find what we want
```

Comment:
@dupirefr was this change intentional?

Reply:
Nope, It must have happened when I refactored the package name. I fixed it a moment ago